### DISCUSSION TITLE
Server side settings

### DISCUSSION THREAD

**Discussion by jackkiej** [OpenRCT2/OpenRCT2#15720] [Feature requests]
I would like a new feature that gives us the ability to create server-side settings.
So everyone that joins the server will play on those settings.

**Comment by Lastorder-DC** [comment]
I don't see any benefit from forcing server's config to client.

**Comment by IntelOrca** [comment]
There are some settings that may cause desyncs, we will ensure those will be the same at some point. But not cosmetic settings like metric or imperial units, that is just a user preference.

**Comment by jackkiej** [comment]
Yeah I didn't mean all the settings ofcourse
